### PR TITLE
DEFAULT GroupType

### PR DESCRIPTION
**Pull request by babenek**
## Description

- Add DEFAULT for GroupType to apply empty filter set (else branch)

## How has this been tested?

- [x] UnitTest

### CODE REVIEW

File: credsweeper/utils/util.py [hunk 1]
```diff
@@ -349,7 +349,7 @@ def is_pdf(data: bytes) -> bool:
 
     @staticmethod
     def is_elf(data: Union[bytes, bytearray]) -> bool:
-        """According https://en.wikipedia.org/wiki/Executable_and_Linkable_Format - use only 5 bytes"""
+        """Accordingly to https://en.wikipedia.org/wiki/Executable_and_Linkable_Format use only 5 bytes"""
```
Comment (csh519) [inline credsweeper/utils/util.py:352]:
```suggestion
        """According to https://en.wikipedia.org/wiki/Executable_and_Linkable_Format use only 5 bytes"""
```

Comment (csh519) [reply credsweeper/utils/util.py:352]: https://textranch.com/340064/according-to/or/accordingly-to/

**Review by csh519 — APPROVED** [review]
Thank you for adding default group.

LGTM :+1:

File: credsweeper/utils/util.py [hunk 2]
```diff
@@ -349,7 +349,7 @@ def is_pdf(data: bytes) -> bool:
 
     @staticmethod
     def is_elf(data: Union[bytes, bytearray]) -> bool:
-        """According https://en.wikipedia.org/wiki/Executable_and_Linkable_Format - use only 5 bytes"""
+        """According to https://en.wikipedia.org/wiki/Executable_and_Linkable_Format use only 5 bytes"""
```
Comment (kmnls) [inline credsweeper/utils/util.py:352]:
```suggestion
        """Accordingly to https://en.wikipedia.org/wiki/Executable_and_Linkable_Format use only 5 bytes"""
```

**Review by kmnls — APPROVED** [review]

**Pull request merged**